api/src/services/observation-service.ts

```diff
@@ -455,9 +471,9 @@ export class ObservationService extends DBService {
           survey_id: surveyId,
           itis_tsn: row['ITIS_TSN'] ?? row['TSN'] ?? row['TAXON'] ?? row['SPECIES'],
           itis_scientific_name: null,
```

Comment:
Do we need to support additional mappings here for itis_scientific_name?
ie: latin_name / scientific_name / species_name

Reply:
Not sure, I'd have to confirm with Macgregor. 

Unless you know of other aliases already in use?

Reply:
I don't think we want to identify species based on scientific name. Although it can be used to identify a taxon, the code is preferred.